public/css/main.css

```diff
@@ -989,7 +989,7 @@ video#bgvid {
     /*left: -30px;*/
 }
 
-.exploremaplink:link {
+.otherCityLinks:link {
```

Comment:
Everything else here looks good. One final nit picky thing: can you make the class name singular instead of plural?